src/Akkling.Persistence/PersistentActor.fs

```diff
@@ -176,6 +177,11 @@ and FunPersistentActor<'Message>(actor : Eventsourced<'Message> -> Effect<'Messa
         | :? LifecycleEvent | :? PersistentLifecycleEvent ->
             // we don't treat unhandled lifecycle events as casual unhandled messages
             current
+        | :? JObject as jobj ->
+            let msg = jobj.ToObject<'Message>()
```

Comment:
Maybe move it right to the `Become(fn) -> ...` branch as it's not used in the other branch?